client/java/src/main/java/com/linecorp/centraldogma/client/AbstractCentralDogmaBuilder.java

```diff
@@ -70,6 +72,8 @@ public abstract class AbstractCentralDogmaBuilder<B extends AbstractCentralDogma
     private int maxNumRetriesOnReplicationLag = DEFAULT_MAX_NUM_RETRIES_ON_REPLICATION_LAG;
     private long retryIntervalOnReplicationLagMillis =
             TimeUnit.SECONDS.toMillis(DEFAULT_RETRY_INTERVAL_ON_REPLICATION_LAG_SECONDS);
+    @Nullable
+    private MeterRegistry meterRegistry;
```

Comment:
By default, it might be a good idea to use `Metrics.globalRegistry`.
```
private MeterRegistry meterRegistry = Metrics.globalRegistry;
```

Reply:
I prefer the default to be null because some people may not be interested in the metrics.